torch_xla/utils/tf_record_reader.py

```diff
@@ -34,28 +34,3 @@ def read_record(self):
       The raw bytes of the record, or ``None`` in case of EOF.
     """
     return torch_xla._XLAC._xla_tfrecord_read(self._reader)
```

Comment:
we can also delete `_xla_tfrecord_read`

Reply:
Thanks, Jack, deleted in commit: https://github.com/pytorch/xla/pull/5112/commits/cefb6602b317fe37756c8cc7865980bf8744dbf4